src/open_inwoner/components/templates/components/Button/Button.html

```diff
@@ -5,8 +5,9 @@
         class="{{ classes }}"
         href="{{href}}"
         title="{% firstof title text %}"
-        aria-describedby="{{ text }}"
+        {% if text %}aria-describedby="{{ text }}"{% endif %}
```

Comment:
> The aria-describedby attribute lists the [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#attr-id)s of the elements that describe the object. It is used to establish a relationship between widgets or groups and the text that describes them.

 I think we should use aria-label here, and text should be mandatory to populate it. We can then choose not to render the text but only the aria-label as the icon is not useful for assistive technology.

Reply:
Applied both changes, using aria-label instead of aria-describedby and removed aria-hidden